### ISSUE TITLE
What's new

### ISSUE THREAD

**Issue by ifrost** [ifrost/afterwriting-labs#58]
List of recent changes

**Comment by owentar** [comment]
Would be nice to have a CHANGELOG

**Comment by ifrost** [comment]
Good point. I've updated Release Notes for recent releases: https://github.com/ifrost/afterwriting-labs/releases
